charts/payment-api/values.yaml

```diff
@@ -72,6 +72,7 @@ java:
     FEATURE_PAYMENTS_SEARCH: "true"
     FEATURE_SERVICE_CALLBACK: "true"
     PAYMENT_S2S_MICROSERVICE_NAME: payment_app
+    PBA_OLD_CONFIG_SERVICE_NAMES: CMC,PROBATE,DIVORCE
```

Comment:
Can we rename to PBA_CONFIG_1_SERVICE_NAMES as we know from Specific Services for Config-1 & Config-2

Reply:
This will go away soon. Config-1 is a bit more clear.

Reply:
I will go with this then **PBA_CONFIG_1_SERVICE_NAMES**